docs/cloud/cli/connector.mdx

```diff
@@ -90,7 +90,6 @@ OPTIONS:
 
 Example usage:
 
-%copy%
```

Comment:
it is weird that in the preview branch of cloudflare I still see this.

Reply:
In the version of the connector I think that we have to put `x.y.z` and the message about checking the version on the hub

Reply:
I replaced the versions with `x.y.z`. Not sure about where to put the message to check version on hub. Maybe we should make a global note somewhere rather than each example.